### DISCUSSION TITLE
Roadmap for v4 stable

### DISCUSSION THREAD

**Discussion by petewalker** [activeadmin/activeadmin#8319] [General]
We're currently using active admin in one of our client projects. We are excited about the changes - particularly around theming - that v4 promises to bring.

We'd like to migrate, but obviously v4 is currently still in beta. Is there a public (or private) roadmap for what's required for v4 to be considered stable?

I had a look around the repo, but it doesn't look like milestones / projects are in use, so I wondered what the process was.

If there are tangible tasks left to do, we'd be happy to contribute!

**Comment by wrozka** [comment]
Same here, I use activeadmin in a private project and I could contribute some of my spare time to help with the release. I'm really excited about the update!

**Comment by petewalker** [comment]
@tagliala - Do you have any information on this?

**Comment by vfonic** [comment]
Came here looking for the same thing.

I have found the v4 beta 1 release note: https://github.com/activeadmin/activeadmin/releases/tag/v4.0.0.beta1

/cc @mgrunberg @javierjulio
